### ISSUE TITLE
Name not registering on completion board

### ISSUE THREAD

**Issue by miguelpotts** [hryanjones/guess-my-word#26]
Hi Hyran.

Guess My Word is great. Love it. 

Found (I think) a bug today. After solving normal, I switched over to hard without registering on the normal completion board. 

Then, after solving hard (10g, 28s!!!) there was no way to register for the hard board. 

I was able to go back and register my normal attempt, but not my hard. 

Using iOS.

**Comment by hryanjones** [comment]
I couldn't reproduce this, could you try the suggestions in this tweet:

https://twitter.com/guessmyword1/status/1233227927757574144

> - try after a "hard" refresh (usually can click on URL and hit enter)
> - try after closing window & browser, then reopen
> - (if not on a phone) you can try to clear your cache (***not cookies/site data as you'll lose game)

**Comment by LisaH415** [comment]
Also Love(!) guess my word. 
And name did not show up on leader board for “Normal” results. 
Oddly enough I see my name on the “Hard” results board (Lisa, 11 guesses, 1m31s). 
Kinda bumming bc I guessed the normal word in one try.

**Comment by hryanjones** [comment]
Yes, this is a bug. Were u using a computer (as opposed to a phone)? If so
I can help u reset the submit with a command in the web console.

**Comment by LisaH415** [comment]
I was using an iPhone

**Comment by hryanjones** [comment]
Darn, well, if you want me to retroactively put your name on the board
(would mostly be useful right now for the all time board stats), then let
me know your:
- username (the one u would've used on the board)
- approximate list of your guesses (please put the final word as the last
one for me to double check)
- approximate time
- whether you want the guesses to be public or not

**Comment by LisaH415** [comment]
Ok. Let’s see - 
- Lisa or Lisa H (usernames used)
- I guessed in one word: “mile” (see screenshot attached)
- Thursday 6/25 @ 1049a
- guesses not public 
Thanks!

![308DFC68-5CAD-43F6-879D-7BB970244793](https://user-images.githubusercontent.com/67437160/86054015-40576800-ba0e-11ea-8d93-8c643c93ef5b.jpeg)
